src/mixxx.cpp

```diff
@@ -261,6 +261,7 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
                              m_pRecordingManager);
     m_pPlayerManager->bindToLibrary(m_pLibrary);
 
+    m_pLibrary->scan();
```

Comment:
As @Be-ing already mentioned the line you added simply duplicates line 391 in the same file, but misses to respect the user's settings. Please imagine that music library directories can get huge and take minutes to rescan.

I recommend to close this PR until you come up with a feasible concept for solving this issue.